askbot/forms.py

```diff
@@ -427,7 +427,7 @@ def __init__(self, *args, **kwargs):
 
 def clean_tag(tag_name, look_in_db=True):
     """a function that cleans a single tag name"""
-    tag_name = unicodedata.normalize('NFC', tag_name)
+    tag_name = slugify(tag_name)
```

Comment:
this I think won't work. Normalization was added to work with compound (e.g. accented) characters.